Automation/Task6/AppGateway.ps1

```diff
@@ -0,0 +1,64 @@
+#Creates Azure Application Gateway 
+
+param (
+    [Parameter(Mandatory=$true)]
+    [string]
+    $applicationGatewayName,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $location,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $backendAddressPools,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $backendHttpSettingsCollection,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $frontendIPConfigurations,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $gatewayIPConfigurations,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $frontendPorts
```

Comment:
There needs to be comma separation between parameters. This is causing issues while running the script.